docs/user-guide/interfaces/cli/basics.md

```diff
@@ -0,0 +1,88 @@
+Before we get to using the CLI, let's cover some information on
+configuration and behavior.
+
+## Configuring
+
+The default Rug CLI configuration file is located at
+`~/.atomist/cli.yml`.  You can override the default location using the
+`--settings=PATH` command-line option.  As the configuration file
+extension suggests, it is a [YAML][yaml] file.
+
+[yaml]: http://yaml.org/
+
+If you are using the default configuration file and one does not
+exist, the CLI will create a default one for you with contents like
+the following.
+
+```yaml
+# Set up the path to the local repository
+local-repository:
+  path: "${user.home}/.atomist/repository"
+
+# Set up remote repositories to query for Rug archives. Additionally one of the
+# repositories can also be enabled for publication (publish: true).
+remote-repositories:
+  maven-central:
+    publish: false
+    url: "http://repo.maven.apache.org/maven2/"
+  rug-types:
+    publish: false
+    url: "https://atomist.jfrog.io/atomist/libs-release"
+  rugs:
+    publish: false
+    url: "https://atomist.jfrog.io/atomist/rugs-release"
+
+# Set up Rug catalog endpoints for searching
+catalogs:
+- "https://api.atomist.com/catalog"
+```
+
+You can configure a repository directory on your local file system
+using the `path` key under the `local-repository` key.  When
+specifying your system home directory, use the portable `${user.home}`
+rather than system-specific `$HOME` or `~`.
+
+You can specify any number of remote repositories under the
+`remote-repositories` key.  The value of the key should be a mapping
+whose keys are unique identifiers for each entry and whose values
+provide the `url` and optionally whether to `publish` rugs to the
+repository.  If you are publishing Rugs to the repository, you will
+likely need to supply `authentication` details for the repository.
+The authentication details should be a mapping specifying the
+`username` and `password`.  A complete configuration for publishing to
+a repository would look something like:
+
+```yaml
+remote-repositories:
+  rugs:
+    url: "https://atomist.jfrog.io/atomist/rugs-release"
+    publish: true
+    authentication:
+      username: "YOUR_USER_NAME"
+      password: "YOUR_PASSWORD_OR_TOKEN"
+```
+
+You can specify any number of Rug catalogs to search using the
+`catalogs` key.  The value of the key should be a sequence where each
+element is a URL to a Rug catalog.
```

Comment:
As we really only have one catalog right now, it is safer to remove that section all together. 

If we want to retain that, it should be change to:

```configuration:
  catalog-service-urls:
  - "https://api.atomist.com/catalog/operation/search"
```